src/libs/ReactNativeLauncher.js

```diff
@@ -4,6 +4,7 @@ import MicroEE from 'microee'
 
 import Launcher from './Launcher'
 import ContentScriptBridge from './bridge/ContentScriptBridge'
+import debounce from 'lodash/debounce'
 
```

Comment:
@doubleface can you fix these warnings 🙏

Reply:
Done